app/presenters/queries/available_translations.rb

```diff
@@ -16,28 +16,58 @@ def translations
       attr_reader :content_id, :with_drafts, :expanded_translations
 
       def grouped_translations
-        Edition.with_document
-          .where("documents.content_id": content_id, state: state_fallback_order)
-          .pluck(:id, "documents.locale", :state)
-          .sort_by { |(_, _, state)| state_fallback_order.index(state.to_sym) }
-          .group_by { |(_, locale)| locale }
+        pluck_and_sort_editions(
+          filter_unpublishings(editions)
+        )
       end
 
       def expand_translation(id)
-        expansion_rules = LinkExpansion::Rules
-        web_item(id).select { |f| expansion_rules.expansion_fields(:available_translations).include?(f) }
+        web_item(id).select do |field|
+          expansion_rules
```

Comment:
might be slightly easier to follow to have `LinkExpansion::Rules` here instead of a method